.travis.yml

```diff
@@ -4,19 +4,25 @@ python:
   - "2.7"
   - "3.6"
 
-env:
-  - TRAVIS=true
-
 install:
  - pip install pytest pytest-cov euclid3 coveralls
 
+
 script:
   - py.test --cov=. tests/
 
 after_success:
   - coverage report
   - coveralls
-  - git config --global user.name "ladybugbot"
-  - git config --global user.email "release@ladybug.tools"
-  - pip install python-semantic-release
-  - semantic-release publish
+
+jobs:
+  include:
+    - stage: deploy
+      if: branch = master AND (NOT type IN (pull_request))
+      python: "3.6"
```

Comment:
@AntoineDao, Is this going to generate the whl file only for Python 3.6 or will it do it for both Python 2 and Python 3?

Reply:
Short answer: I have no idea... Haven't dived that deep into python packaging. As far as I know it was initially only deploying using one python version anyways (either 2.7 or 3.6 depending on which finished testing and deploying first) and I have managed to run dragonfly on both python 2 and python 3 without any issues.

Reply:
from https://pythonwheels.com/

>Note: If your project is Python 2 and 3 compatible you can create a universal wheel distribution. Create a file called setup.cfg with the following content and upload your package.
```
[bdist_wheel]
universal = 1
```

Reply:
Nice!